haunted-houses/src/main/java/cz/muni/fi/pa165/hauntedhouses/dao/impl/HouseDaoImpl.java

```diff
@@ -1,6 +1,96 @@
 package cz.muni.fi.pa165.hauntedhouses.dao.impl;
 
 import cz.muni.fi.pa165.hauntedhouses.dao.HouseDao;
+import cz.muni.fi.pa165.hauntedhouses.model.House;
 
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.NoResultException;
+import javax.persistence.Persistence;
+import java.util.List;
+
+/**
+ * @author Petr Vitovsky
+ */
 public class HouseDaoImpl implements HouseDao {
+    private static EntityManagerFactory emFactory = Persistence.createEntityManagerFactory("default");
+
+    @Override
+    public void create(House house) {
+        EntityManager em = emFactory.createEntityManager();
+        em.getTransaction().begin();
+
+        em.persist(house);
+
+        em.getTransaction().commit();
+        em.close();
+    }
+
+    @Override
+    public House readId(long id) {
+        EntityManager em = emFactory.createEntityManager();
+        em.getTransaction().begin();
+
+        House house = em.find(House.class, id);
+
+        em.getTransaction().commit();
+        em.close();
+        return house;
+    }
+
+    @Override
+    public House readAddress(String address) {
```

Comment:
Ja by som navrhoval premenovat funkciu na nieco ako getHouseByAddress (to iste aj pre readId -> getHouseById a pre readAll -> getAllHouses). Tie nazvy nevystihuju to, co tie funkcie robia. readAddress znie, ako keby to malo precitat nejaku adresu.

Reply:
Souhlasím, přejmenováno.